internal/server/placepage/golden/get_place_page_data/county.sample.json

```diff
@@ -24270,7 +24342,7 @@
           "metadata": {
             "importName": "NASA_NEXDCP30_StatVarSeriesAggr",
             "provenanceUrl": "https://www.nccs.nasa.gov/services/data-collections/land-based-products/nex-dcp30",
-            "measurementMethod": "dcAggregate/NASA_Mean_CCSM4",
```

Comment:
This change was made a while back and I also remember seeing similar diff before... so I don't understand t his one now..

Reply:
See the file below. The base cache mode was commented out so the change was not applied before.